Onboarding/menu-non-personalised.md

```diff
@@ -0,0 +1,687 @@
+<!--
+ dictionary: "config"
+version: "0.1.0"
+columns: [] 
+-->
+
+| Key               | Value                          |
+| ----------------- |--------------------------------|
+| contentrepo_token | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+
+```stack
+card FetchError, then: NonPersonalisedMenu do
+  # Fetch and store the error message, so that we don't need to do it for every error card
+
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_error_handling_button"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
```

Comment:
Instead of `@config.items.contentrepo_token` you can now use `@global.config.contentrepo_token` and then you don't have to worry about the dictionary and token in every Journey anymore.

Reply:
Also, when we revoke the token, we don't have to update every Journey file.

Reply:
Hi @HawkiesZA last time when i changed to global variable it was not working. So my suggestion will be to work on these changes later on all journeys please.

Reply:
@Buhle79 There are flows that use it (e.g. Basic Questions, HCW Profile, Intro & Welcome), so it definitely works. The one mistake that I've made before is to only change `@config` to `@global`, which won't work, you have to replace `@config.items` with @global.config`. Maybe it's that?

Reply:
@Buhle79 Agreed though, we can park it for now. Let's get everything else done.

Reply:
Cool thanks, can you please approve this PR.